docs/provision_hosted_cluster_on_mce_local_cluster.md

```diff
@@ -0,0 +1,288 @@
+- `region`: Region of the S3 bucket
+
+See [Getting started](https://hypershift-docs.netlify.app/getting-started). in the HyperShift documentation for more information about the secret. The following example shows a sample AWS secret creation CLI template:
+
+```bash
+$ oc create secret generic hypershift-operator-oidc-provider-s3-credentials --from-file=credentials=$HOME/.aws/credentials --from-literal=bucket=<s3-bucket-for-hypershift> --from-literal=region=<region> -n local-cluster
+```
+
+**Note:** Disaster recovery backup for the secret is not automatically enabled. Run the following command to add the label that enables the `hypershift-operator-oidc-provider-s3-credentials` secret to be backed up for disaster recovery:
+
+```bash
+$ oc label secret hypershift-operator-oidc-provider-s3-credentials -n local-cluster cluster.open-cluster-management.io/backup=true
+```
+
+#### Enabling External DNS
+If you plan to use service-level DNS for Control Plane Service, create an external DNS credential secret for the HyperShift operator, and name it `hypershift-operator-external-dns-credentials`. It should reside in the managed cluster namespace (i.e., the namespace of the managed cluster that will be used as the hosting service cluster). If you used `local-cluster`, then create the secret in the `local-cluster` namespace
+
+The secret must contain 3 fields:
+
+- `provider`: DNS provider that manages the service-level DNS zone (example: aws)
+- `domain-filter`: The service-level domain
+- `credentials`: *(Optional, only when using aws keys) - For all external DNS types, a credential file is supported
+- `aws-access-key-id`: *OPTIONAL* - When using AWS DNS service, credential access key id
+- `aws-secret-access-key`: *OPTIONAL* - When using AWS DNS service, credential access key secret
+
+For details, please check: [HyperShift Project Documentation](https://hypershift-docs.netlify.app/how-to/external-dns/). For convenience, you can create this secret using the CLI by:
+
+```bash
+$ oc create secret generic hypershift-operator-external-dns-credentials --from-literal=provider=aws --from-literal=domain-filter=service.my.domain.com --from-file=credentials=<credentials-file> -n local-cluster
+```
+
+Add the special label to the `hypershift-operator-external-dns-credentials` secret so that the secret is backed up for disaster recovery.
+
+```bash
+$ oc label secret hypershift-operator-external-dns-credentials -n local-cluster cluster.open-cluster-management.io/backup=true
+```
+
+##### Enable on a HostedCluster
+
+1. Install the HyperShift add-on. The cluster that hosts the HyperShift operator is the management cluster. This step uses the hypershift-addon to install the HyperShift operator on a managed cluster. `ManagedClusterAddon` hypershift-addon. The namespace will be the managed cluster on which you want to install the HyperShift operator on. In this case, we will use the MCE hub cluster, so we'll set `local-cluster` for this value:
```

Comment:
This step uses the hypershift-addon to install the HyperShift operator on a managed cluster. `ManagedClusterAddon` hypershift-addon.

How about change to:

This step uses the `ManagedClusterAddon` hypershift-addon to install the HyperShift operator on a managed cluster.